src/Microsoft.AspNet.WebHooks.Receivers.Azure/Microsoft.AspNet.WebHooks.Receivers.Azure.nuspec

```diff
@@ -6,12 +6,14 @@
     <title>Microsoft ASP.NET WebHooks Receiver for Azure WebHooks.</title>
     <authors>Microsoft</authors>
     <owners>Microsoft, aspnet</owners>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
-    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=690277</projectUrl>
-    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <licenseUrl>https://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <projectUrl>https://go.microsoft.com/fwlink/?LinkId=690277</projectUrl>
+    <iconUrl>https://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>
-      This package provides support for receiving WebHooks from Kudu (Azure Web App Deployment) and Azure Alerts. For information about Kudu WebHooks, see "https://github.com/projectkudu/kudu/wiki/Web-hooks" and for Azure Alerts, see "https://azure.microsoft.com/en-us/documentation/articles/insights-webhooks-alerts/".
+      This package provides support for receiving WebHooks from Kudu (Azure Web App Deployment) and Azure Alerts. For
+      information about Kudu WebHooks, see https://github.com/projectkudu/kudu/wiki/Web-hooks and for Azure Alerts, see
+      https://azure.microsoft.com/en-us/documentation/articles/insights-webhooks-alerts/.
```

Comment:
BTW whether this shows up correctly on MyGet is unlikely to be a good indication of whether this looks good on NuGet.org. The code for it is here: https://github.com/NuGet/NuGetGallery/blob/da86aef01bf5bfbdaedac486137d0019b75dcdbe/src/NuGetGallery/Helpers/HtmlExtensions.cs#L54

It would be prudent to just try that code on this input and see what it generates.

Reply:
Thanks very much for the pointer. We're good.

Tried it out (using our new friend the VS C# Interactive window 😺) with the following sample:
``` c#
var text = " https://azure.microsoft.com/en-us/documentation/articles/insights-webhooks-alerts/.";
```
Result was just lovely:
``` c#
" <a href=\"https://azure.microsoft.com/en-us/documentation/articles/insights-webhooks-alerts/\" rel=\"nofollow\">https://azure.microsoft.com/en-us/documentation/articles/insights-webhooks-alerts/</a>."
```

Reply:
Nice!